src/patternfly/components/Page/page.scss

```diff
@@ -321,6 +322,13 @@ $pf-page-v6--height-breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg
   background: var(--#{$page}__main-container--BackgroundColor);
   border: var(--#{$page}__main-container--BorderWidth) solid var(--#{$page}__main-container--BorderColor);
   border-radius: var(--#{$page}__main-container--BorderRadius);
+
+  &.pf-m-fill,
+  &:has(.#{$page}__main-section.pf-m-fill,
+    .#{$page}__main-group.pf-m-fill,
+    .#{$page}__main-wizard.pf-m-fill) {
```

Comment:
I suspect we will want the main content to fill if:
* There is a wizard in the page body area at all (it wouldn't necessarily need `.pf-m-fill`, it would just fill by default). It was a goal in v5 for the wizard footer to remain static regardless the height of the wizard content, and the wizard designs for v6 look the same. We got that by having the wizard fill the main container area and set the wizard content to scroll.
* There is a drawer attached to the main container. In that case, I'd probably expect the `__main-container` to fill but no page sections fill by default - page sections will still look like they normally do, but the drawer panel can expand over the full height of the page content area. Assuming that's still what we want in v6, that could be a reason to not make page sections fill the height by default.

Reply:
OK - I changed it so that any wizard will make the main container fill, and a drawer inside will as well - bottom drawer won't though. WDYT?